### PR TITLE
feat: add issue templates with spec scaffold (bug, feature, task)

### LINKED ISSUE #120: feat: GitHub issue templates with spec scaffold (bug, feature, task)
## Problem

\`.github/ISSUE_TEMPLATE/\` has only \`pulse-feedback.md\`. No templates for
bug/feature/task → humans open issues without structure → AC, out of scope,
NFRs, edge cases absent from the start.

Even after #119 trains the detailing agent, brainstorming-stage issues arrive
blank → agent must infer structure instead of filling scaffold.

## Sprint Goal

Same as #119 — both issues together move all 6 dimensions to 10/10.
This issue covers the human entry point; #119 covers the agent output.

| Dimension | Before | Target |
|---|---|---|
| Out of scope | 1/10 | 10/10 |
| Given/When/Then | 3/10 | 10/10 |
| NFRs com número | 1/10 | 10/10 |
| Success metrics em features | 2/10 | 10/10 |
| Edge cases sistemáticos | 3/10 | 10/10 |
| AC em features/enhancements | 2/10 | 10/10 |

Measured on: next 5 non-bug issues created after both PRs merge.

## Solution

Create 3 issue templates with opinionated scaffolding. Brainstorming sections
are free-form (human fills). Detailing sections are pre-labeled
`<!-- agent fills during detailing -->` so agent knows what to complete.

## Acceptance Criteria

**AC1 — bug.md template exists**
- Given user opens new issue on GitHub
- When user selects "Bug" template
- Then form pre-fills: Problem, Root Cause, Reproduction Steps, ACs (Given/When/Then), Out of Scope, NFRs, Files to Modify

**AC2 — feature.md template exists**
- Given user opens new issue on GitHub
- When user selects "Feature" template
- Then form pre-fills: Problem, Sprint Goal/Metrics, Solution (blank), ACs (Given/When/Then), Edge Cases, Out of Scope, NFRs, Files to Modify

**AC3 — task.md template exists**
- Given user opens new issue on GitHub
- When user selects "Task" template
- Then form pre-fills: Objective, Steps, ACs (checklist), Out of Scope, Files to Modify

**AC4 — templates replicated in templates/ for downstream projects**
- Given `npx create-agentic-pdlc` scaffolds a new project
- When `.github/ISSUE_TEMPLATE/` is created
- Then bug.md, feature.md, task.md are present with same scaffolding

## Edge Cases

- EC1: User opens issue without selecting template → GitHub default blank form → acceptable, no solution needed
- EC2: Template has `<!-- comment -->` sections → invisible to human on GitHub UI but visible in raw markdown → agent reads raw → correct behavior
- EC3: Downstream project has existing ISSUE_TEMPLATE files → `npx create-agentic-pdlc --update` must not overwrite → out of scope for this issue

## Out of Scope

- Form-based templates (GitHub `issue-forms` YAML) — markdown templates sufficient, less complexity
- Auto-labeling via template `labels:` frontmatter — label automation owned by existing workflows
- Validating required fields before submission — GitHub doesn't support required fields in markdown templates

## Non-Functional Requirements

- Template size: each file < 80 lines — longer templates are ignored by humans
- All comment placeholders use `<!-- agent fills during detailing -->` — consistent signal for agent

## Files to Modify

- `.github/ISSUE_TEMPLATE/bug.md` — create
- `.github/ISSUE_TEMPLATE/feature.md` — create
- `.github/ISSUE_TEMPLATE/task.md` — create
- `templates/.github/ISSUE_TEMPLATE/bug.md` — create (downstream scaffold)
- `templates/.github/ISSUE_TEMPLATE/feature.md` — create
- `templates/.github/ISSUE_TEMPLATE/task.md` — create

### PR DESCRIPTION
**Pull request by rafaeltcosta86**
## Summary

- Creates `.github/ISSUE_TEMPLATE/` templates for bug, feature, and task issue types
- Each template scaffolds all 7 spec sections: Problem, Sprint Goal, Solution, ACs (Given/When/Then), Edge Cases, Out of Scope, NFRs
- Replicates templates in `templates/.github/ISSUE_TEMPLATE/` for downstream `npx create-agentic-pdlc` projects

## Test plan

- [ ] Open new issue on GitHub → verify 3 templates appear in selector
- [ ] Select Feature → confirm all 7 sections present with `<!-- agent fills during detailing -->` markers
- [ ] Select Bug → confirm Root Cause + Reproduction Steps + ACs present
- [ ] Select Task → confirm checklist-style ACs + Out of Scope present
- [ ] Verify `templates/.github/ISSUE_TEMPLATE/` has all 3 files

Closes #120

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+name: Feature
+about: New capability with spec scaffold for detailing agent
+title: 'feat: '
+labels: enhancement
```
[the remark this answers is not in the document]
Comment (rafaeltcosta86) [reply .github/ISSUE_TEMPLATE/feature.md]: fixed.

File: templates/.github/ISSUE_TEMPLATE/feature.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+name: Feature
+about: New capability with spec scaffold for detailing agent
+title: 'feat: '
+labels: enhancement
```
[the remark this answers is not in the document]
Comment (rafaeltcosta86) [reply templates/.github/ISSUE_TEMPLATE/feature.md]: FIxed.
